Quiz.py

```diff
@@ -0,0 +1,85 @@
+print("Hello, welcome to Jaz's Quiz")
+
+ans = input("Are you ready to play ? (yes / no): ")
+score = 0
+total_questions = 10
+
+
+if ans.lower() == 'yes':
+    ans = input('1. What is the capital of the United Kingdom ? ')
+    if ans.lower() == 'london':
+        score += 1
+        print("Correct")
+    else:
+        print("Incorrect")
+
+    ans = input('2. Who is the longest-reigning monarch in British history ? ')
+    if ans.lower() == 'the queen' or 'queen':
+        score += 1
+        print("Correct")
+    else:
+        print("Incorrect")
+
+    ans = int(input('3. What is 5 multiplied by 5 ? '))
+    if ans == 25:
+        score += 1
+        print("Correct")
+    else:
+        print("Incorrect")
+
+    ans = input('4. What is the most common colour of toilet paper in France ? ')
+    if ans.lower() == 'pink':
+        score += 1
+        print("Correct")
+    else:
+        print("Incorrect")
+
+    ans = int(input('5. How many days are there is February ? '))
+    if ans == 28:
+        score += 1
+        print("Correct")
+    else:
+        print("Incorrect")
+        ans = int(input('5. How many days are there is February ? '))

```

Comment:
Why do you save the answer here?

Reply:
this would be a mistake of repetition from my end. commented line out completely